Sources/Passes/Passes.docc/Personalization.md

```diff
@@ -68,13 +77,14 @@ final class PassDelegate: PassesDelegate {
 
     func encodePersonalization<P: PassModel>(for pass: P, db: any Database, encoder: JSONEncoder) async throws -> Data? {
         guard let passData = try await PassData.query(on: db)
-            .filter(\.$pass.$id == pass.id!)
+            .filter(\.$pass.$id == pass.requireID())
+            .with(\.$pass)
             .first()
```

Comment:
Can we just query on `Pass` instead of `PassData` here? Or even `UserPersonalization`

Reply:
To query `UserPersonalization` I think that you'll need joins, since the `@OptionalChild` property is not enforced by the protocol. Anyways, this is just a suggestion to the user that in the delegate they could have access to `PassData` if they need it and that they should check if a pass' `userPersonalization` exists to be sure that the personalization process was completed.

Reply:
To query `UserPersonalization` I think that you'll need joins, since the `@OptionalChild` property is not enforced by the protocol. Anyways, this is just a suggestion to the user that in the delegate they have access to `PassData` if they need it and that they should check if a pass' `userPersonalization` exists to be sure that the personalization process was completed.